airflow/api_connexion/openapi/v1.yaml

```diff
@@ -2161,6 +2161,32 @@ paths:
         "404":
           $ref: "#/components/responses/NotFound"
 
+  /dagStats:
+    get:
+      summary: List Dag statistics
+      x-openapi-router-controller: airflow.api_connexion.endpoints.dag_stats_endpoint
+      operationId: get_dag_stats
+      tags: [DagStats]
+      parameters:
+        - name: dag_ids
+          in: query
+          schema:
+            type: string
+          required: true
+          description: |
+            One or more DAG IDs separated by commas to filter relevant Dags.
```

Comment:
Thanks so much for working on this! One question I have here is if this can be an optional parameter, and by default, return all the stats of all the dags, which is the current behavior of the `dag_stats` endpoint.

Reply:
Good point. This also allows us to add other filter parameters like how the Dags list has active/paused DAGs

Reply:
Thanks for the feedback! Of course I can set it to optional. But in my opinion we then also need to handle the `limit` and `offset` parameters to reduce the potentially large query result set when you have many different DAGs and DAG runs. 

The current implementation returns the statistics of all selected DAGs through the request_form. Thus it is implicitly limited [here](https://github.com/apache/airflow/blob/92c8697f20d97b5bd1eae2142412d18bc972906d/airflow/www/views.py#L1229).

Reply:
I could still see us wanting to make the dag_ids optional and adding pagination. But we can add that later without affecting backcompat.